src/main/java/org/myrobotlab/swing/AndroidSpeechRecognitionGui.java

```diff
@@ -31,8 +32,8 @@
   public final static Logger log = LoggerFactory.getLogger(_TemplateServiceGui.class);
 
   private JTextField onRecognized = new JTextField("Waiting orders...", 24);
-  BufferedImage microOn = ImageIO.read(FileIO.class.getResource(Util.getResourceDir() + "/InMoov/monitor/microOn.png"));
-  BufferedImage microOff = ImageIO.read(FileIO.class.getResource(Util.getResourceDir() + "/InMoov/monitor/microOff.png"));
+  BufferedImage microOn = ImageIO.read(new File(Util.getResourceDir() + "/InMoov/monitor/microOn.png"));

```

Comment:
This will only work in an IDE. I suspect it would still fail if this is running on windows as getResourceDir() obeys the proper File.sep